cpu/esp32/periph/adc.c

```diff
@@ -149,14 +149,6 @@ const gpio_t _gpio_rtcio_map[] = {
     RTCIO_SENSOR_SENSE4, /* GPIO39 SENSOR_VN */
 };
 
-/** Map of RIOT ADC and DAC lines to GPIOs */
-static const uint32_t adc_pins[] = ADC_GPIOS;
-static const uint32_t dac_pins[] = DAC_GPIOS;
```

Comment:
I think that the DAC pins removal belongs to #11290, otherwise compile error arise.

Reply:
Yes, you are right. I just missed it when I added the part of `adc.c` while splitting the PRs.

Reply:
Fixed it in this PR and PR #11290.